packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -9,12 +14,41 @@ jsf.option({
   optionalsProbability: 1,
   fixedProbabilities: true,
   ignoreMissingRefs: true,
-  useExamplesValue: true,
-  useDefaultValue: true,
   maxItems: 20,
   maxLength: 100,
 });
 
-export async function generate(source: unknown): Promise<unknown> {
-  return jsf.resolve(cloneDeep(source));
+export function generate(source: JSONSchema): unknown {
+  return jsf.generate(cloneDeep(source));
```

Comment:
What's the difference here? `generate` vs `resolve`?

Reply:
`generate` is the sync-version, with partial dereferencing through given refs — the idea is that Graphite will resolve all the references for us so we do not need to go through them again.